PGM/src/main/java/tc/oc/pgm/polls/commands/PollSubCommands.java

```diff
@@ -94,34 +105,31 @@ public void pollKick(CommandContext args, CommandSender sender) throws CommandEx
             return CommandUtils.completeMapName(mapName);
         }
 
-        if (!Config.Poll.enabled()) {
+        if(!Config.Poll.enabled()) {
```

Comment:
Can you let the poll package have its own configuration class? I think it would be better than referencing it from the Config class.